pkgs/by-name/yo/you-have-mail-cli/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  llvmPackages,
+  libclang,
+  go,
+  buildGoModule,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "you-have-mail-cli";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "LeanderBB";
+    repo = "you-have-mail-cli";
+    rev = "b338153353495428e2dc0065843553b7821b9d3a";
+    hash = "sha256-YgIztCmL+uJUUXu+BWXLf9f4L7UpuPf0onsfMRtlWkw=";
+  };
+
+  nativeBuildInputs = [
+    libclang.lib
+    llvmPackages.libcxxClang
+    go
+  ];
+
+  LIBCLANG_PATH = "${libclang.lib}/lib";
```

Comment:
You may want to add `rustPlatform.bindgenHook` to `nativeBuildInputs` and remove all things about clang.